Dockerfile-scraper

```diff
@@ -0,0 +1,8 @@
+FROM alpine:latest
```

Comment:
By the `-scraper` suffix I understand this is for KSM, right?

Reply:
Ok, I remembered now, it will be the same image for all scrapers. Why jsut not call it `nri-kubernetes` then? The other one will just be the `forwarder`.